app/Database/migrations/2024_11_16_105923_create_simbrief_support_tables.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        if (!Schema::hasTable('simbrief_aircraft')) {
+            Schema::create('simbrief_aircraft', function (Blueprint $table) {
+                $table->increments('id');
+                $table->string('icao');
+                $table->string('name');
+                $table->mediumText('details')->nullable();
+                $table->timestamps();
+            });
+        }
+
+        if (!Schema::hasTable('simbrief_airframes')) {
+            Schema::create('simbrief_airframes', function (Blueprint $table) {
+                $table->increments('id');
+                $table->string('icao');
+                $table->string('name');
+                $table->string('airframe_id')->nullable();
+                $table->string('source')->nullable();
```

Comment:
Why not use an int and an enum like `AirframeSource` instead, and then do something like `AirframeSource::CUSTOM` or `AirframeSource::SIMBRIEF`? 

This would avoid relying on "magic" strings

Reply:
Thought about it, actually was considering INTERNAL/EXTERNAL kind of difference or just 0/1 without any strings.

But was lazy enough to return back to there, we can do it.